### ISSUE TITLE
Adjust Collection/UseConstraints

### ISSUE THREAD

**Issue by jmw0057** [NASA-IMPACT/pyQuARC#71]
**Describe the bug**
Collection/UseConstraints only accepts whitespace as an entry in the test file. An error is produced when anything other than white space is input into the field.

**To Reproduce**
1. Add "UseConstraints" as a field in the test file.
2. Input random text.
3. See error "Character content other than whitespace is not allowed because the content type is 'element-only'."

**Expected behavior**
Did not expect to see error when inputting random characters.

**Outputs**

**Additional context**

**Comment by slesaad** [comment]
I cannot reproduce this error. It lets me add any random characters. I don't know if this was solved in earlier commits, but can you check again @jmw0057 ?

**Comment by jmw0057** [comment]
I'm still seeing the same error when using the updated pyQuARC code. It's possible that I may have tested something incorrectly on my end, but I've attached some pictures for further reference. I added the 'UseConstraints' field after the 'DOI' field in the 'test_cmr_metadata.echo10" file, and then ran the code.

![Added_Use_Constraints](https://user-images.githubusercontent.com/57103986/115780025-4b54fd80-a37e-11eb-8d72-65e412dcb4f5.PNG)

![Error_message](https://user-images.githubusercontent.com/57103986/115780109-61fb5480-a37e-11eb-803d-c6f72bdd1bb2.PNG)
